rollup.config.common.js

```diff
@@ -0,0 +1,34 @@
+import resolve from '@rollup/plugin-node-resolve';
+import json from '@rollup/plugin-json';
+import babel from '@rollup/plugin-babel';
+import peerDepsExternal from 'rollup-plugin-peer-deps-external';
+
+export default {
+  cache: false,
+  // external: [
+  //   'react',
+  //   'react-dom',
+  //   '@ima/core',
+  // ],
+  input: 'src/main.js',
+  output: {
+    file: 'dist/main.js',
+    format: 'cjs'
+  },
+  treeshake: false,

```

Comment:
Why false?

Reply:
Need to be discussed: https://rollupjs.org/guide/en/#treeshake

Reply:
I think, that this is one of the reasons we are moving to rollup, so I would set it to true, or remove this option since true is the default value.